packages/in_app_purchase/in_app_purchase_storekit/example/ios/RunnerTests/FIATransactionCacheTests.m

```diff
@@ -38,19 +38,14 @@ - (void)testGetObjectsForNonExistingKey {
   XCTAssertNil([cache getObjectsForKey:TransactionCacheKeyUpdatedTransactions]);
 }
 
-- (void)testRemoveObjectsForNonExistingKey {
-  FIATransactionCache *cache = [[FIATransactionCache alloc] init];
-  [cache removeObjectsForKey:TransactionCacheKeyUpdatedTransactions];
-}
-
-- (void)testRemoveObjectsForExistingKey {
+- (void)testClear {
   NSArray *dummyArray = @[ @1, @2, @3 ];
   FIATransactionCache *cache = [[FIATransactionCache alloc] init];
   [cache addObjects:dummyArray forKey:TransactionCacheKeyUpdatedTransactions];
 
   XCTAssertEqual(dummyArray, [cache getObjectsForKey:TransactionCacheKeyUpdatedTransactions]);
 
-  [cache removeObjectsForKey:TransactionCacheKeyUpdatedTransactions];
+  [cache clear];
   XCTAssertNil([cache getObjectsForKey:TransactionCacheKeyUpdatedTransactions]);
```

Comment:
Just to be safe, it would be good to add something for each key before, and assert that they are all nil after.

Reply:
You are right, should have done so right away.